### ISSUE TITLE
ETag value on record enpoint: timestamp of record or collection ?

### ISSUE THREAD

**Issue by leplatrem** [mozilla-services/cliquet#356]
```
     $ http GET https://kinto.dev.mozaws.net/v1/buckets/default/collections/tasks/records \
            --auth 'user:password'
            -v --auth 'user:password'
     GET /v1/buckets/default/collections/tasks/records HTTP/1.1
     Accept: */*
     Accept-Encoding: gzip, deflate
     Authorization: Basic dXNlcjpwYXNzd29yZA==
     Connection: keep-alive
     Host: kinto.dev.mozaws.net
     User-Agent: HTTPie/0.9.2
 
     HTTP/1.1 200 OK
     Access-Control-Expose-Headers: Backoff, Retry-After, Alert, Next-Page, Total-Records, Last-Modified, ETag
     Backoff: 10
     Connection: keep-alive
     Content-Length: 152
     Content-Type: application/json; charset=UTF-8
     Date: Thu, 18 Jun 2015 15:34:04 GMT
     ETag: "1434641474977"
     Last-Modified: Thu, 18 Jun 2015 15:31:14 GMT
     Server: nginx/1.4.6 (Ubuntu)
     Total-Records: 1
 
     {
         "data": [
             {
                 "description": "Write a tutorial explaining Kinto",
                 "id": "23eed462-c063-4ae0-81b0-8bf2210bfe86",
                 "last_modified": 1434641515332,
                 "status": "todo"
             }
         ]
     }
```

I would have expected : 
* On record enpoints, the ETag is the value of the record timestamp (ie. ``last modified``)
* On collection endpoints, the ETag is the value of the collection timestamp

**Comment by Natim** [comment]
This should be fixed with 1.2 it was one of the bugs fixed with ``get_parent_id``

**Comment by leplatrem** [comment]
It's a matter of specs. In the code, we add the collection timestamp on records response headers.

I just confirmed it on the Kinto server.

```
$ echo '{"data": {"title":1}}' | http POST "https://kinto.dev.mozaws.net/v1/buckets/default/collections/tasks/records" --auth 'user:password'


$ echo '{"data": {"title":2}}' | http POST "https://kinto.dev.mozaws.net/v1/buckets/default/collections/tasks/records" --auth 'user:password'

$ http "https://kinto.dev.mozaws.net/v1/buckets/default/collections/tasks/records/6c5f255d-65cf-4eec-a0ee-4c1132b19f15" --auth 'user:password'
HTTP/1.1 200 OK
Access-Control-Expose-Headers: Backoff, Retry-After, Alert, Last-Modified, ETag
Backoff: 10
Connection: keep-alive
Content-Length: 197
Content-Type: application/json; charset=UTF-8
Date: Mon, 06 Jul 2015 08:41:47 GMT
ETag: "1436172070772"
Last-Modified: Mon, 06 Jul 2015 08:41:10 GMT
Server: nginx/1.4.6 (Ubuntu)

{
    "data": {
        "id": "6c5f255d-65cf-4eec-a0ee-4c1132b19f15", 
        "last_modified": 1436172059952, 
        "title": 1
    }, 
    "permissions": {
        "write": [
            "basicauth:10ea4e5fbf849196a4fe8a9c250b737dd5ef17abbeb8f99692d62828465a9823"
        ]
    }
}
```
